### PR TITLE
any: performance improvement and bugfix

### PR DESCRIPTION
**Pull request by mikecovlee**

### CODE REVIEW

File: shared/include/covscript/shared/any.hpp [hunk 1]
```diff
@@ -142,7 +132,7 @@ class alignas(alignof(std::size_t)) cs::runtime::any final {
 	{
 		switch (m_data.status) {
 		case stor_status::null:
-			return nullptr;
+			throw_ex<runtime_error>("Access null any object.");
```
Comment (imkiva) [inline shared/include/covscript/shared/any.hpp]: Use `cs::runtime_error`. This is quite confusing with `std::runtime_error`.

File: shared/include/covscript/shared/any.hpp [hunk 2]
```diff
@@ -155,7 +145,7 @@ class alignas(alignof(std::size_t)) cs::runtime::any final {
...
```
Comment (imkiva) [inline shared/include/covscript/shared/any.hpp]: Same as above.

File: shared/include/covscript/shared/any.hpp [hunk 3]
```diff
@@ -9,12 +9,12 @@ namespace cs {
...
+class cs::runtime::any final {
 public:
 	using typeid_t = std::type_index;
 	using byte_t = unsigned char;
 	// 缓冲池大小，过大的值可能会适而其反
-	static constexpr std::size_t default_allocate_buffer_size = 64;
+	static constexpr std::size_t default_allocate_buffer_size = 16;
 	/*
 	* 分配器提供者，默认使用STL分配器，可根据需要替换为内存池
 	* 提示：本框架的Any使用了Small Data Optimize技术，可大幅减少堆的负担
```
Comment (imkiva) [inline shared/include/covscript/shared/any.hpp]: Spaces in comments.

File: shared/include/covscript/shared/any.hpp [hunk 4]
```diff
@@ -58,10 +58,9 @@ class alignas(alignof(std::size_t)) cs::runtime::any final {
...
 	class stor_impl : public stor_base {
-		// 实际存储的数据
-		T m_data;
-
 	public:
+		// 实际存储的数据
+		T data;
 		// 分配器
 		static default_allocator<stor_impl<T>> allocator;
 		// 默认构造函数，使用default实现
```
Comment (imkiva) [inline shared/include/covscript/shared/any.hpp]: Spaces in comments

**Review by imkiva — COMMENTED** [review]
Coding style.

**Review by imkiva — CHANGES_REQUESTED** [review]
Coding style.

**Comment by mikecovlee** [comment]
Finished.

File: shared/include/covscript/shared/any.hpp [hunk 5]
```diff
@@ -282,24 +272,22 @@ class alignas(alignof(std::size_t)) cs::runtime::any final {
 
 	// 提取数据方法封装
 	template <typename T>
-	T &get()
+	inline T &get()
 	{
-		if (m_data.status == stor_status::null)
-			throw_ex<runtime_error>("Access null any object.");
```
Comment (imkiva) [inline shared/include/covscript/shared/any.hpp:288]: `cs::runtime_error`

File: shared/include/covscript/shared/any.hpp [hunk 6]
```diff
@@ -282,24 +272,22 @@ class alignas(alignof(std::size_t)) cs::runtime::any final {
...
-		if (get_handler()->type() != typeid(T))
+		stor_base *ptr = get_handler();
+		if (ptr->type() != typeid(T))
 			throw_ex<runtime_error>("Access wrong type of any.");
```
Comment (imkiva) [inline shared/include/covscript/shared/any.hpp]: `cs::runtime_error`

File: shared/include/covscript/shared/any.hpp [hunk 7]
```diff
@@ -282,24 +272,22 @@ class alignas(alignof(std::size_t)) cs::runtime::any final {
...
 	template <typename T>
-	const T &get() const
+	inline const T &get() const
 	{
-		if (m_data.status == stor_status::null)
-			throw_ex<runtime_error>("Access null any object.");
-		if (get_handler()->type() != typeid(T))
+		const stor_base *ptr = get_handler();
+		if (ptr->type() != typeid(T))
 			throw_ex<runtime_error>("Access wrong type of any.");
```
Comment (imkiva) [inline shared/include/covscript/shared/any.hpp]: `cs::runtime_error`

**Review by imkiva — CHANGES_REQUESTED** [review]
My fault. I didn't review code carefully enough.///

**Comment by mikecovlee** [comment]
Merge master changes and replace all of `runtime_error` to `cs::runtime_error`

**Pull request merged**